Simplified/NYPLBookDetailButtonsView.h

```diff
@@ -0,0 +1,26 @@
+#import "NYPLBookButtonsView.h"
+
+@class NYPLBook;
+@class NYPLBookDetailButtonsView;
+@class NYPLBookDetailDownloadFailedView;
+
+@protocol NYPLBookDetailDownloadingDelegate
+
+- (void)didSelectCancelForBookDetailDownloadingView:
+(NYPLBookDetailButtonsView *)bookDetailButtonsView;
+
+- (void)didSelectCancelForBookDetailDownloadFailedView:
+(NYPLBookDetailButtonsView *)NYPLBookDetailDownloadFailedView;
+
+@end
+
+@interface NYPLBookDetailButtonsView : UIView
+
+@property (nonatomic, weak) NYPLBook *book;
```

Comment:
Given that an `NYPLBook` will never hold a reference to an `NYPLBookDetailButtonsView`, I am not sure why this reference should be `weak`.